docs/stencil-docs/getting-started/transitioning-to-stencil.md

```diff
@@ -5,149 +5,88 @@
 
-As with Blueprint themes, Stencil themes can receive updates from theme developers. You can apply theme updates by going to the BigCommerce Control Panel. When in the control panel, navigate to **Storefront** › **My Themes**, and click **Update**.
+If you are still using our legacy theme framework, Blueprint, we highly encourage switching to Stencil so you can take advantage of the new features and benefits. For more in-depth information about Stencil’s features, see the [Stencil Theme Platform](https://support.bigcommerce.com/s/article/The-Stencil-Theme-Platform#features) page.
 
-BigCommerce shares updates to Cornerstone in our Changelog (https://developer.bigcommerce.com/changelog) so you can verify that updates don't conflict with your customizations.
+## Why Transition to Stencil?
 
-For more on theme updates and best practices for handling version control, view this guide:
-https://developer.bigcommerce.com/stencil-docs/getting-started/advanced-installation-options/theme-updates-and-version-control#theme-updates_goruck-best-pracs
+Stencil improves the overall storefront experience for merchants and their customers. With enhancements in speed, security, and updates to your themes, this modern framework provides a more streamlined process for the everyday maintenance and tasks required to run a store.
 
-### FAQ
+Stencil themes are built on non-proprietary, open-source code to provide easier adoption and adaptation while providing greater flexibility and access for customization. In comparison, Blueprint, is built on a rigid, proprietary codebase, making it difficult to tailor and extend theme functionality.
 
-**How will this affect my store? Will I lose data?**
+For more information on the enhanced capabilities of Stencil, see the [Stencil vs. Legacy Blueprint](https://support.bigcommerce.com/s/article/The-Stencil-Theme-Platform#compare) page.
 
-No store data will be lost when you switch from Blueprint to Stencil. A store's theme is the presentation layer that renders your store catalog. You will lose any information you've hard-coded into your previous Blueprint theme.
+## Technical Differences
 
-**How can I retain assets from my Blueprint theme?**
+When transitioning any store from Blueprint to Stencil, you should be aware of a few key differences before you begin. Transitioning to Stencil allows developers to do the following:
+- [Develop locally on Stencil CLI]()
+- [Use Handlebars instead of global variables]()
+- [Customize Store Design mode]()
+- [Create custom templates]()

```

Comment:
Did you add these links in a later commit? Didn't see them

Reply:
Yes, I added these in a subsequent commit